### DISCUSSION TITLE
Other phonemizers and support for other OSALR voices

### DISCUSSION THREAD

**Discussion by luiscarlos2000** [rhasspy/piper#197] [Ideas]
Hi comunity,
Someone asked it on an issue I've recently commented, but is eSpeakNG the only supporter phonemyzer on Piper_ If possible could you implement support for others such as DeepPhonemizer or something? I forgot the real name of it...
Also, browsing through all the OSALR (Open Speech and Language Resources), I've discovered many speech corpuses on many languages, mine (Spanish) included, and the already mentioned/discussed LibriTTS R set. If I want to mention the SLR identifiers I will, however not this time since work.
I appreciate all the answers.

**Comment by luiscarlos2000** [comment]
And FYW, the lynk of the resources in question:
http://www.openslr.org/resources.php

**Comment by synesthesiam** [comment]
Hello! espeak-ng is the only real phonemizer currently supported (the other is raw text). DeepPhonemizer could work, but it would need to be exported to onnx.

For datasets, many of the voices come from OpenSLR resources. I'm actually training a LibriTTS-R voice right now 🙂

**Reply by luiscarlos2000** [reply]
Ah I see. Because those resources are very usefull, even there's spanish voices there if I or someone can train all of them. SLR ID's 61, 71, 72, 73, 74, 75 are in majority latin american (Argentinian, Chilean, Colombian, Peruvian, Puerto-Rican and Venezuelan) spanish. I believe there's already some spanish language voices from the Multilingual LibriSpeech (MLS) dataset. A friend of mine on another forum told or asked if you can add those voices. I believe it's a lot of work, but yeah, you know, more than ever means mor view.
